include/common_package/common_node.hpp

```diff
@@ -5,9 +5,12 @@
 #include <functional>
 #include <memory>
 #include <string>
-
+#include <nlohmann/json.hpp>
```

Comment:
Linebreaks kosten nix. Ich weiß, welch Ironie, dass das gerade von mir kommt.😅
```suggestion
#include <nlohmann/json.hpp>

```

Reply:
Ich versuche aktuell folgendes System bei Headern zu verfolgen:

```Libraries
[Leerzeile]
Custom Libraries bzw. Node eigene .hpp Dateien
[Leerzeile]
Message Includes```

Dementsprechend ist auch diese Datei (hoffentlich 😅) formatiert.